fury/tests/test_actors.py

```diff
@@ -1355,3 +1355,24 @@ def test_billboard_actor(interactive=False):
     scene.add(actor.axes())
     if interactive:
         window.show(scene)
+
+
+def test_sdf_actor(interactive=False):
```

Comment:
can you add the following decorator at the top of the function. there is an example upper in this file. (you will have to import `skip_win` at the top of the file):

``` 
@pytest.mark.skipif(skip_win, reason="This test does not work on Windows "
                                                                 " due to snapshot (memory access violation)"
                                                                 "Check what is causing this issue with shader")
```